lydia-braga/1.ii/cv.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Currículo</title>
+</head>
+<body>
+    <img src="lydia.jpg" alt="Foto de perfil">
+
+    <h1>Lydia de Castro Cançado Braga</h1>
+    <address>
+        Instituto Nacional de Telecomunicações - Av. João de Camargo, 510, 37540-000, Santa Rita do Sapucaí -MG<br>
+        Telefone (35) 3471-2719 - Celular (35) 98845-2719<br>
+        lydia.braga@inatel.br
+    </address>
+
+    <section>
+        <h2>Resumo Profissional</h2>
+        <p>Atuação na área de Tecnologia da Informação como desenvolvedora de software,
+    com experiência principal na linguagem Java e desenvolvimento de backend.
+    Familiar com Framework Spring e seus principais conceitos, serviços RESTful e bancos de dados tradicionais e NoSQL.
+    Possui conhecimento em testes unitários como forma de assegurar a qualidade dos sistemas desenvolvidos.</p>
+        <p>Conhecimento inicial em desenvolvimento Web com Javascript e Framework AngularJS.</p>
+    </section>
+
+    <section>
+        <h2>Experiência</h2>
+
+        <h3>Especialista em Sistemas</h3>
+        <small>
+            Inatel<br>
+            Janeiro de 2016 - Presente
+        </small>
```

Comment:
This tag was really difficult to choose. I tried many others until I got to this one.
The goal of this text is to be a small detail under the title, so it is not a subtitle, neither a quotation or a paragraph.
Looking at the mdn documentation, I found out that small is used to represent side-comments and small print, so it was the closest I could get to what I wanted to represent.

Do you know another tag that fits better for this situation?